client/src/app/shared/components/Notifications.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import {
+  Alert,
+  AlertActionCloseButton,
+  AlertGroup,
+} from "@patternfly/react-core";
+import { NotificationsContext } from "../notifications-context";
+
+export type notificationType =
```

Comment:
Instead of duplicating the string union here, we can remove the `notificationType` type and instead import `AlertProps` from PatternFly, and use `AlertProps['variant']` as the type below.

Either that, or we could use `export interface INotification extends Pick<AlertProps, 'variant'> {`, both have the same effect.

Reply:
Nice !
I went with first approach. 
Thanks